plug-ins/prometheus/test-setup.md

```diff
@@ -0,0 +1,32 @@
+# Prometheus Plugin Test Setup
+
+* Install Prometeus 
+  * wget https://github.com/prometheus/prometheus/releases/download/v2.42.0/prometheus-2.42.0.linux-amd64.tar.gz
```

Comment:
Is there a known minimum Prometheus version required?  The packaged version in Ubuntu 22 is 2.31.